src/connector.js

```diff
@@ -0,0 +1,612 @@
+				resultLink.onclick = ( e ) => { interactiveResult.select(); };
+				resultLink.oncontextmenu = ( e ) => { interactiveResult.select(); };
+				resultLink.onmousedown = ( e ) => { interactiveResult.select(); };
+				resultLink.onmouseup = ( e ) => { interactiveResult.select(); };
+				resultLink.ontouchstart = ( e ) => { interactiveResult.beginDelayedSelect(); };
+				resultLink.ontouchend = ( e ) => { interactiveResult.cancelPendingSelect(); };
+
+				resultListElement.appendChild( sectionNode );
+			});
+		}
+		else if ( resultListState.firstSearchExecuted && !resultListState.hasResults && !resultListState.hasError ) {
+			resultListElement.innerHTML = noResultTemplateHTML;
+		}
+	}
+}
+
+// Update heading that has number of results displayed
+function updateQuerySummaryState( newState ) {
+	querySummaryState = newState;
+	
+	if ( resultListState.firstSearchExecuted && !querySummaryState.isLoading && !querySummaryState.hasError ) {
+		querySummaryElement.textContent = "";
+		if ( querySummaryState.total > 0 ) {
+			let numberOfResults = querySummaryState.total.toString();
+
+			if ( numberOfResults.length > 3 ) {
+                numberOfResults = numberOfResults.substring( 0, numberOfResults.length - 3 ) + ',' + numberOfResults.substring( numberOfResults.length - 3 );
+            }
+			if ( numberOfResults.length > 7 ) {
+                numberOfResults = numberOfResults.substring( 0, numberOfResults.length - 7 ) + ',' + numberOfResults.substring( numberOfResults.length - 7 );
+            }
+
+            querySummaryElement.innerHTML = ( ( querySummaryState.query != "" && !params.isAdvancedSearch ) ? querySummaryTemplateHTML : noQuerySummaryTemplateHTML )
+				.replace( '%[numberOfResults]', numberOfResults )
+				.replace( '%[query]', querySummaryState.query );
+		}
+	}
+	else if ( querySummaryState.hasError ) {
+		querySummaryElement.textContent = "";
+		querySummaryElement.innerHTML = resultErrorTemplateHTML;
+	}
+}
+
+// Update pagination
+function updatePagerState( newState ) {
+	pagerState = newState;
+	pagerElement.textContent = "";
+	let innerHTML = "";
+	if ( pagerState.hasPreviousPage ) {
+		const liNode = document.createElement( "li" );
+		liNode.innerHTML = previousPageTemplateHTML;
+		const aNode= liNode.querySelector( 'a' );
+		aNode.onclick = ( e ) => { 
+            pagerController.previousPage();
+            window.scrollTo( { top: 200, behavior: 'smooth' } );
+        };
+		pagerElement.appendChild( liNode );
+	}
+	pagerState.currentPages.forEach( ( page ) => {
+		const liNode = document.createElement( "li" );
+		if ( page == pagerState.currentPage ) {
+			liNode.setAttribute( "class","active" );
+		}
+		const pageNo = page;
+		liNode.innerHTML = pageTemplateHTML.replace( '%[page]', pageNo );
+		const aNode = liNode.querySelector( 'a' );
+		aNode.onclick = ( e ) => {
+            pagerController.selectPage( pageNo );
+            window.scrollTo( { top: 200, behavior: 'smooth' } );
```

Comment:
it seem those pagination are acting like a button, please code them by using the button element instead of the anchor element.

Reply:
Agreed this needs to be revisited. This is a note for @bblaisATcoveo for his subsequent PR